src/ui/components/EventsCalendarTeaser.tsx

```diff
@@ -21,9 +21,9 @@ export function EventsCalendarTeaser(props: Props) {
       {loading && !events ? (
         <LoadingBox />
       ) : topThreeEvents?.length > 0 ? (
-        topThreeEvents.map((event, id) => (
-          <React.Fragment key={id}>
-            <View style={styles.container}>
+        topThreeEvents.map((event) => (
+          <React.Fragment key={event.id}>
+            <View style={styles.container} testID="upcoming_event_items">
```

Comment:
One thing you can do the know what to query for in a test instead of going for the testID is to use the debug helper: 

```
const { debug } = render(<Component />);
debug();
```
It will print the component tree